hydra-core/lib/generators/hydra/templates/config/fcrepo_wrapper_development.yml

```diff
@@ -0,0 +1,4 @@
+#config/fcrepo_wrapper_development.yml
+port: 8984
+enable_jms: false
+fcrepo_home_dir: tmp/fcrepo4-development-data
```

Comment:
If we want to be nice to downstream users, can we add a trailing new line to these yml files?